dashboard/src/components/widget/footer/che-footer.directive.ts

```diff
@@ -87,7 +87,7 @@ export class CheFooter {
     }
 
     // docs
-    template += '<a class=\"che-footer-button-blue che-footer-button\" href=\"/docs\" target=\"_blank\">Docs<a/>';
+    template += '<a class=\"che-footer-button-blue che-footer-button\" href=\"/docs/factory/getting-started/index.html\" target=\"_blank\">Docs<a/>';
```

Comment:
@olexii4 is it okay from your point of view, that link to all docs now points to factories section?

https://github.com/eclipse/che/blob/master/dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html#L179
https://github.com/eclipse/che/blob/master/dashboard/src/app/factories/list-factories/list-factories.html#L21